client/src/styles/theme.scss

```diff
@@ -1,8 +1,8 @@
 // Import Colors
 @import "./colors.scss";
-@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap");
```

Comment:
didn't we also have an npm dependency for the Poppins font?

Reply:
Yea it's called `typeface-poppins`. Before adding the weight values the bold text were off so not sure if the dependency works ...

Reply:
I'm glad to see @anniegiang using the @import for the google fonts as I've just added Work Sans and DM Sans to the index.scss file - I'll move it to theme.scss. I'll remove poppins dependency in my work for #88